android/app/src/main/java/com/expensify/chat/MainApplication.java

```diff
@@ -66,6 +67,7 @@ public void onCreate() {
       super.onCreate();
       // If you opted-in for the New Architecture, we enable the TurboModule system
       ReactFeatureFlags.useTurboModules = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED;
+      ReactFontManager.getInstance().addCustomFont(this, "Expensify New Kansas", R.font.expensifynewkansas);
```

Comment:
@luacmartins Why did you add the font in this way? This would be the only xml font in the application. I think we should really stick with the pattern established for other native fonts here:

- put a native `.otf` font in `/assets/fonts/native`
- Run `npx react-native-asset – that will:
    - Add the native font to `android/link-assets-manifest.json`
    - Put a duplicate font in `android/app/src/main/assets/fonts` instead of in `android/res/font`

Reply:
Then no native code should be required to load the font

Reply:
I explained a bit of it [here](https://github.com/Expensify/App/pull/13714#discussion_r1054817989). Is there another way to add it and be able to use `Expensify New Kansas` as `fontFamily` across all platforms?

Based on [this conversation](https://github.com/Expensify/App/pull/13714#discussion_r1053792219), it seems like @Luke9389 will work on a follow up PR to rename the other fonts to use the correct fontFamily name as well. So we'd solve any inconsistencies there.

Reply:
As discussed 1:1, let's revert that change from this PR and then create a follow-up issue to address the XML fontFamily thingy. I'd love to work on a way to make that improvement upstream without compromising the ease of font asset management we get from https://github.com/unimonkiez/react-native-asset